app/models/public_comment.rb

```diff
@@ -1,30 +1,32 @@
 class PublicComment < Comment
 
-  after_create :create_notifications, :send_emails
+  after_create :notify, :send_emails
```

Comment:
remove the :send_emails callback since notify can directly call that itself.  Whether it's in app or via email, they'll both a form of notifications so conceptually notify should handle both.